.github/workflows/publish-npm-packages.yml

```diff
@@ -48,11 +48,56 @@ jobs:
         run: |
           EXTENSIONS=$(node ${{ env.SCRIPTS_DIR }}/get-extensions.js)
           echo "::set-output name=extensions::{\"include\":${EXTENSIONS}}"
-  publish:
+  publish-core:
     if: github.repository == 'ampproject/amphtml'
     environment: bento_publishing
     needs: setup
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
@estherkim should this be false?

Reply:
yes, this allows packages to be published independently. If true, say bento-accordion fails, the other jobs will be aborted.